programs/helium-entity-manager/src/state.rs

```diff
@@ -10,6 +10,10 @@ pub struct HotspotConfigV0 {
   pub authority: Pubkey,
   pub symbol: String,
   pub sub_dao: Pubkey,
+  pub min_gain: i32,
+  pub max_gain: i32,
+  pub full_location_staking_fee: u64,
```

Comment:
`staking_fee` is throwing me off. Should it just be `dc_fee`? are they actually staking something?

Reply:
It's a misnomer but that's the term used by Helium